test/cors.spec.js

```diff
@@ -0,0 +1,20 @@
+let helper = require('./helper');
+
+describe('CORS', function () {
+  it('GET /stories should allow cross origin', function(done) {
+    helper.request()
+          .get('/stories')
+          .end(function(err, res) {
+            res.should.to.have.header('Access-Control-Allow-Origin');
```

Comment:
Not sure. In development, you usually set `Access-Control-Allow-Origin` to `*` but it can (and maybe should) be limited to specific domain for security.